lib/components/ModalBase/style.scss

```diff
@@ -58,11 +58,10 @@
 	display: flex;
 	align-content: flex-end;
 	justify-content: center;
-	align-items: flex-end;
-	height: 100%;
+	height: auto;
+	align-items: baseline;
```

Comment:
I think we should still keep this as "align-items: flex-end. Guess the only reason it works not is that there is no room for movement between containers.